backend/apps/matching/migrations/0002_permission_group_creation.py

```diff
@@ -0,0 +1,143 @@
+# Generated by Django 3.0.7 on 2020-06-15 12:28
+
+from django.db import migrations
+
+
+# This is added here and not in the project because these names are know to be valid only
```

Comment:
What is meant by "not in the project"? I am unfamiliar with the migrations. Can we go over them?

Reply:
Let's discuss it in the next call